src/tests/ftest/soak/faults.yaml

```
@@ -72,18 +72,18 @@ container_reserved:
     control_method: daos
 faults:
     fault_list:
-    #    - DAOS_DTX_LOST_RPC_REQUEST
-    #    - DAOS_DTX_LOST_RPC_REPLY
-    #    - DAOS_DTX_LONG_TIME_RESEND
-       - DAOS_SHARD_OBJ_UPDATE_TIMEOUT
-       - DAOS_SHARD_OBJ_FETCH_TIMEOUT
-    #    - DAOS_SHARD_OBJ_FAIL
-    #    - DAOS_OBJ_UPDATE_NOSPACE
-    #    - DAOS_SHARD_OBJ_RW_CRT_ERROR
-    #    - DAOS_OBJ_REQ_CREATE_TIMEOUT
-    #    - DAOS_SHARD_OBJ_UPDATE_TIMEOUT_SINGLE
-    #    - DAOS_OBJ_SPECIAL_SHARD
-    #    - DAOS_OBJ_TGT_IDX_CHANGE
+        #    - DAOS_DTX_LOST_RPC_REQUEST
+        #    - DAOS_DTX_LOST_RPC_REPLY
+        #    - DAOS_DTX_LONG_TIME_RESEND
+        - DAOS_SHARD_OBJ_UPDATE_TIMEOUT
+        - DAOS_SHARD_OBJ_FETCH_TIMEOUT
+        #    - DAOS_SHARD_OBJ_FAIL
+        #    - DAOS_OBJ_UPDATE_NOSPACE
+        #    - DAOS_SHARD_OBJ_RW_CRT_ERROR
+        #    - DAOS_OBJ_REQ_CREATE_TIMEOUT
+        #    - DAOS_SHARD_OBJ_UPDATE_TIMEOUT_SINGLE
+        #    - DAOS_OBJ_SPECIAL_SHARD
+        #    - DAOS_OBJ_TGT_IDX_CHANGE
```

This is nit picking, but with this change uncommenting the lines via `Ctrl + /` in Visual Studio Code ends up requiring additional work as opposed to the original.  This would be better:
```suggestion
        # - DAOS_DTX_LOST_RPC_REQUEST
        # - DAOS_DTX_LOST_RPC_REPLY
        # - DAOS_DTX_LONG_TIME_RESEND
        - DAOS_SHARD_OBJ_UPDATE_TIMEOUT
        - DAOS_SHARD_OBJ_FETCH_TIMEOUT
        # - DAOS_SHARD_OBJ_FAIL
        # - DAOS_OBJ_UPDATE_NOSPACE
        # - DAOS_SHARD_OBJ_RW_CRT_ERROR
        # - DAOS_OBJ_REQ_CREATE_TIMEOUT
        # - DAOS_SHARD_OBJ_UPDATE_TIMEOUT_SINGLE
        # - DAOS_OBJ_SPECIAL_SHARD
        # - DAOS_OBJ_TGT_IDX_CHANGE
```

This would also be consistent with commented out lines in src/tests/ftest/soak/harassers.yaml and src/tests/ftest/soak/smoke.yaml

---

Will do.